oauthlib/oauth2/rfc6749/clients/service_application.py

```diff
@@ -68,6 +68,7 @@ def prepare_request_body(self,
                              audience=None,
                              expires_at=None,
                              issued_at=None,
+                             extra_jwt_headers=None,
```

Comment:
is it needed for any specific specification or for a particular API/use case only?

Reply:
Standard header parameters are defined in [RFC 7515](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1). In this document, `"kid"` is defined, but also other parameters that may be meaningfully used here.